public/app/features/dashboard-scene/sharing/types.ts

```diff
@@ -1,5 +1,13 @@
-import { SceneObject, SceneObjectState } from '@grafana/scenes';
+import { SceneObject, SceneObjectRef, SceneObjectState } from '@grafana/scenes';
 
-export interface SceneShareTab<T extends SceneObjectState = SceneObjectState> extends SceneObject<T> {
+export interface ModalSceneObjectLike {
+  onDismiss: () => void;
+}
+
+export interface SceneShareTabState extends SceneObjectState {
+  modalRef?: SceneObjectRef<ModalSceneObjectLike>;
```

Comment:
Creating this interface for tabs so that we can pass ref to the modal and interact with it, i.e. via Cancel button that dismisses the modal